vacasa/connect/requests_config.py

```diff
@@ -0,0 +1,40 @@
+import random
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+RETRY_STATUS_CODES = (500, 502, 503, 504)
+
+
+class BackoffWithJitter:
+    """urllib3.util.retry.Retry expects a float for `backoff_factor`, but we want to sneak in some jitter """
+
+    def __init__(self, backoff, jitter_min, jitter_max):
+        self._backoff = backoff
+        self._jitter = (jitter_min, jitter_max)
+
+    def __mul__(self, other):
+        """Add some random jitter every time the backoff function tries to do some multiplication"""
+        return (self._backoff + random.uniform(*self._jitter)) * other
```

Comment:
What do you think of this solution?  
https://github.com/abrookins/reliable_service_calls_with_python/blob/master/simulation/jittery_retry.py

Reply:
That's way better, thanks!